inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -98,6 +99,7 @@ class LSTMCellStage final : public StageNode {
 
 static void RNNRelayout(
```

Comment:
Why it's static? We can use anonimous namespace for local entities

Reply:
Actually, there should not be changes in this method at all. Let us leave it without changes for now.